Apps/Admin/Client/Store/Dashboard/DashboardActions.cs

```diff
@@ -47,35 +47,45 @@ public LoadRegisteredUsersAction(int timeOffset)
     }
 
     /// <summary>
-    /// The action representing the initiation of logged in users load action.
+    /// The action representing a successful registered user action.
     /// </summary>
-    public class LoadLoggedInUsersAction
+    public class RegisteredUsersSuccessAction : BaseSuccessAction<IDictionary<DateTime, int>>
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="LoadLoggedInUsersAction"/> class.
+        /// Initializes a new instance of the <see cref="RegisteredUsersSuccessAction"/> class.
         /// </summary>
-        /// <param name="timeOffset">The offset from the client browser to UTC.</param>
-        public LoadLoggedInUsersAction(int timeOffset)
+        /// <param name="data">user data.</param>
+        public RegisteredUsersSuccessAction(IDictionary<DateTime, int> data)
+            : base(data)
         {
-            this.TimeOffset = timeOffset;
         }
+    }
 
+    /// <summary>
+    /// The action representing a failed registered user action.
+    /// </summary>
+    public class RegisteredUserFailAction : BaseFailAction
```

Comment:
Typo: In this action, `RegisteredUser` should be `RegisteredUsers` to match the related actions.